hack/verify-all.sh

```diff
@@ -18,11 +18,7 @@ set -euo pipefail
 
 readonly PKG_ROOT="$(git rev-parse --show-toplevel)"
 
-${PKG_ROOT}/hack/verify-gofmt.sh
-${PKG_ROOT}/hack/verify-govet.sh
-${PKG_ROOT}/hack/verify-golint.sh
 ${PKG_ROOT}/hack/verify-yamllint.sh
 ${PKG_ROOT}/hack/verify-boilerplate.sh
-${PKG_ROOT}/hack/verify-spelling.sh
```

Comment:
let's keep the original script, it's still useful in local dev, dev could run `hack/verify-all.sh` before submitting PR,

Reply:
Okay, shall we remove `make verify` from the workflow then? It would just run the same checks twice

Reply:
let's keep it for a period of time, finally should remove the check in `pull-csi-driver-smb-verify`, `pull-csi-driver-smb-unit` and travis test. Need stabilization time.

Reply:
Thanks @andyzhangx 
I have added the required linters to `golangci-lint` action.

> let's keep it for a period of time, finally should remove the check in pull-csi-driver-smb-verify, pull-csi-driver-smb-unit and travis test. Need stabilization time.

Shall I open an issue for this so that it is kept track of?